### PR TITLE
BXMSDOC-4646-master: Update managing projects in business central for DM/PAM doc per changes in 7.5

### PR DESCRIPTION
**Pull request by kaldesai**
[Epic](https://issues.jboss.org/browse/BXMSDOC-4408)
[JIRA](https://issues.jboss.org/browse/BXMSDOC-4646)
[Managing projects in Business Central-7.5-RHPAM](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4646-RHPAM-MP-Final/#idm140047208529008)
[Managing projects in Business Central-7.5-RHDM](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4646-RHDM-MP-Final/#idm140520471435152)

Can you please review and verify the managing projects in business central document preview links as per 7.5? 
I have updated the screen-shots in **chapter 4** and **section 8.3**.
Thank you.

### CODE REVIEW

File: doc-content/enterprise-only/project-data/delete-branches-proc.adoc [hunk 1]
```diff
@@ -7,6 +7,6 @@ You can delete any branch except for the *master* branch. {CENTRAL} does not all
 . Click image:project-data/dots.png[] in the upper-right corner of the screen and select *Delete Branch*.
 +
 .Deleting a branch menu
-image::getting-started/delete-branch-menu.png[Delete a branch menu]
```
Comment (pefernan) [inline doc-content/enterprise-only/project-data/delete-branches-proc.adoc]: Could you please also remove the  _delete-branch-menu.png_ file? (assuming it's no longer needed)

File: doc-content/enterprise-only/project-data/new-project-proc.adoc [hunk 2]
```diff
@@ -9,6 +9,6 @@ A project can contain data models, business processes, business rules, forms or
 . Type `anyName` in the *Name* field of the *Add Project* window and click *Add*. Where `anyName` is any name that you want to name the new project.
 +
 .Add the `anyName` project
-image::getting-started/new-proj.png[Adding the `anyName` project window]
```
Comment (pefernan) [inline doc-content/enterprise-only/project-data/new-project-proc.adoc]: Same for  _new-proj.png_ file

File: doc-content/enterprise-only/project-data/delete-branches-proc.adoc [hunk 3]
```diff
@@ -7,6 +7,6 @@ You can delete any branch except for the *master* branch. {CENTRAL} does not all
 . Click image:project-data/dots.png[] in the upper-right corner of the screen and select *Delete Branch*.
 +
 .Deleting a branch menu
```
Comment (csherrar) [inline doc-content/enterprise-only/project-data/delete-branches-proc.adoc]: Minor question - is the title "Deleting a branch menu" correct? I see the image file was renamed to `delete-branch-menu.png`.

File: doc-content/enterprise-only/project-data/delete-branches-proc.adoc [hunk 4]
```diff
@@ -7,6 +7,6 @@ You can delete any branch except for the *master* branch. {CENTRAL} does not all
...
-image::getting-started/delete-branch-menu.png[Delete a branch menu]
+image::getting-started/delete-branch-option.png[Delete a branch menu]
 
 . Input the branch name to be deleted in the *Delete Branch* window and click *Delete Branch*.
```
Comment (csherrar) [inline doc-content/enterprise-only/project-data/delete-branches-proc.adoc]: Minor suggestion - confirm that "to be" is ok. I see we use more instances of "you want".

For example:
. "In the *Delete Branch* window, enter the name of the branch you want to delete."
. "Click *Delete Branch*.

File: doc-content/enterprise-only/project-data/delete-branches-proc.adoc (hunk 3 above)
Comment (kaldesai) [reply doc-content/enterprise-only/project-data/delete-branches-proc.adoc]: Sorted

**Pull request merged**